pkg/util/proto/document.go

```diff
@@ -56,18 +56,31 @@ func VendorExtensionToMap(e []*openapi_v2.NamedAny) map[string]interface{} {
 	return values
 }
 
-// Definitions is an implementation of `Models`. It looks for
+// definitions is an implementation of `Models`. It looks for
```

Comment:
why unexport this?

Reply:
I shouldn't have exported it in the first place ... Now I don't think anything builds on that, and if it does, it should stop anyway.